pallets/ajuna-awesome-avatars/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+authors     = [ "Ajuna Network <https://github.com/ajuna-network>" ]
+description = "Ajuna Network pallet board pallet"
+edition     = "2021"
+homepage    = "https://ajuna.io"
+license     = "Apache-2.0"
+name        = "pallet-ajuna-awesome-avatars"
+repository  = "https://github.com/ajuna-network/Ajuna"
+version     = "0.1.0"
+
+[package.metadata.docs.rs]
+targets = [ "x86_64-unknown-linux-gnu" ]
+
+[dependencies]
+ajuna-common = { default-features = false, path = "../../ajuna-common" }
+codec        = { default-features = false, features = [ "derive", "max-encoded-len" ], package = "parity-scale-codec", version = "3.0.0" }
+scale-info   = { default-features = false, features = [ "derive" ], version = "2.1" }
+
+#frame-benchmarking = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.20", optional = true }
```

Comment:
Maybe benchmarking can be left in as it is only built in when we activate this feature